frontend/src/component/environments/EnvironmentTable/EnvironmentActionCell/EnvironmentCloneModal/EnvironmentCloneModal.tsx

```diff
@@ -0,0 +1,385 @@
+                        </StyledInputDescription>
+                        <EnvironmentProjectSelect
+                            projects={projects}
+                            setProjects={setProjects}
+                        />
+                        <StyledInputDescription>
+                            Keep the users permission to this environment?
+                        </StyledInputDescription>
+                        <StyledInputSecondaryDescription>
+                            If you turn it off, the permission for this
+                            environment across all projects and feature toggles
+                            will remain only for admin and editor roles.
+                        </StyledInputSecondaryDescription>
+                        <FormControlLabel
+                            label={clonePermissions ? 'Yes' : 'No'}
+                            control={
+                                <Switch
+                                    onChange={e =>
+                                        setClonePermissions(e.target.checked)
+                                    }
+                                    checked={clonePermissions}
+                                />
+                            }
+                        />
+                        <StyledSecondaryContainer>
+                            <StyledInputDescription>
+                                API Token
+                            </StyledInputDescription>
+                            <StyledInputSecondaryDescription>
+                                In order to connect your SDKs to your newly
+                                cloned environment, you will also need an API
+                                token.{' '}
+                                <Link
+                                    href="https://docs.getunleash.io/reference/api-tokens-and-client-keys"
+                                    target="_blank"
+                                >
+                                    Read more about API tokens
+                                </Link>
+                                .
+                            </StyledInputSecondaryDescription>
+                            <FormControl>
+                                <RadioGroup
+                                    value={apiTokenGeneration}
+                                    onChange={e =>
+                                        setApiTokenGeneration(
+                                            e.target.value as APITokenGeneration
+                                        )
+                                    }
+                                    name="api-token-generation"
+                                >
+                                    <FormControlLabel
+                                        value={APITokenGeneration.LATER}
+                                        control={<Radio />}
+                                        label="I want to generate an API token later"
```

Comment:
Optional but this keeps this as the same style as the option below it

```suggestion
                                        label="Generate an API token later"
```